README.md

```diff
@@ -1,11 +1,44 @@
 # Eclipse Adoptium
 
-The Adoptium Working Group promotes and supports high-quality runtimes and associated technology for use across the Java ecosystem.
-Our vision is to meet the needs of Eclipse and the broader Java community by providing runtimes for Java-based applications.
-We embrace existing standards and a wide variety of hardware and cloud platforms.
+This organization provides a home for Git repositories that implement of the Adoptium Working Group, The Eclipse Adoptium Top Level Project and several Eclipse projects that fall under that top level project:
+
+* The [Eclipse Adoptium top level project](https://projects.eclipse.org/projects/adoptium).
+* The [Eclipse Adoptium incubator project](https://projects.eclipse.org/projects/adoptium.incubator).
+* The [Eclipse AQAvit project](https://projects.eclipse.org/projects/adoptium.aqavit).
+* The [Eclipse Temurin project](https://projects.eclipse.org/projects/adoptium.temurin).
+* The [Eclipse Temurin Compliance project](https://projects.eclipse.org/projects/adoptium.temurin-compliance).
+* The [Eclipse Mission Control project](https://projects.eclipse.org/projects/adoptium.mc).
 
-This organization provides a home for Git repositories that implement of the Adoptium Working Group and the Eclipse Adoptium top level project.
+**NOTE:** The high-level tracking for the Eclipse Temurin binaries is in the [adoptium repo issue tracking](https://www.github.com/adoptium/adoptium/issues).
+
+----
 
 Please see the [Eclipse Adoptium Project](https://projects.eclipse.org/projects/adoptium) for more information regarding the Adoptium top-level project or its sub-projects (visually depicted in the diagram below).
 
-![Extra Projects](images/AdoptiumCoreRepoRelationshipsDiagram.png)
\ No newline at end of file
+![Extra Projects](images/AdoptiumCoreRepoRelationshipsDiagram.png)
+
+## Eclipse Adoptium Working Group
+
+The Adoptium Working Group promotes and supports high-quality runtimes and associated technology for use across the Java ecosystem.
+Our vision is to meet the needs of Eclipse and the broader Java community by providing a marketplace for high-quality Java runtimes for
+Java-based applications. We embrace existing standards and a wide variety of hardware and cloud platforms.
+
+## Eclipse Adoptium Top Level Project
+
+The mission of the Eclipse Adoptium Top-Level Project is to produce high-quality runtimes and associated technology for use within the Java ecosystem.  We achieve this through a set of Projects under the Adoptium PMC and a close working partnership with external projects, most notably OpenJDK for providing the Java SE runtime implementation. Our goal is to meet the needs of both the Eclipse community and broader runtime users by providing a comprehensive set of technologies around runtimes for Java applications that operate alongside existing standards, infrastructures, and cloud platforms.
```

Comment:
Perhaps "distribute" rather then "produce" when we consider the marketplace?

Reply:
Updated.